stencil.config.ts

```diff
@@ -34,7 +34,10 @@ export const config: Config = {
     { type: "docs-readme" },
     {
       type: "www",
-      copy: [{ src: "../demos", dest: "demos" }],
+      copy: [
+        { src: "../demos", dest: "demos" },
+        { src: "../../node_modules/@esri/calcite-components/dist", dest: "node_modules/@esri/calcite-components/dist" }
```

Comment:
Do we need to copy over the entire dist folder? We probably don't need the loader, types, esm-es5 and maybe less. Could you try just coyping over `../../node_modules/@esri/calcite-components/dist/calcite` and see if it's depending on any of the other directory content?